tests/unit/package_managers/yarn/test_main.py

```diff
@@ -463,10 +463,13 @@ def test_fetch_yarn_source(
     ]
     mock_project_from_source_dir.assert_has_calls(calls)
 
+    from hermeto.core.constants import Mode
```

Comment:
This one should be either on module level or have a comment explaining why it is here.